crates/rolldown/src/module_loader/module_loader.rs

```diff
@@ -220,7 +239,13 @@ impl ModuleLoader {
             .into_iter()
             .zip(resolved_deps)
             .map(|(raw_rec, info)| {
-              let id = self.try_spawn_new_task(&info);
+              let id = self.try_spawn_new_task(
+                &info,
+                Some(ImporterRecord {
+                  kind: raw_rec.kind,
+                  importer_path: module.resource_id.expect_file().clone(),
```

Comment:
A module could have multiple importers. Why we do we want to use this one?

Reply:
See above.